src/Java.Interop.Export/Java.Interop/MarshalMemberBuilder.cs

```diff
@@ -370,7 +370,12 @@ static Expression GetRuntime ()
 			return Expression.Property (null, typeof (JniEnvironment), "Runtime");
 		}
 
-		static  MethodInfo  FormatterServices_GetUninitializedObject    = Type.GetType ("System.Runtime.Serialization.FormatterServices", throwOnError: true)
+		static  MethodInfo  FormatterServices_GetUninitializedObject    =
+#if NETFRAMEWORK || NET_2_0
```

Comment:
`#if NETCOREAPP` might be simpler here (plus switch the branches). Nothing important though.

Reply:
When we start targeting .NET 5/6, won't we also need `NET`? Meaning we'd likewise want `#if NETCOREAPP || NET`?  Which doesn't actually seem simpler…

Reply:
net5, net6, ... also define `NETCOREAPP`